functions/go-mnq-sqs-publish/handler.go

```diff
@@ -25,7 +25,7 @@ func init() {
 
 	sqsSession := session.Must(session.NewSession(&aws.Config{
 		Region:      aws.String("fr-par"),
-		Endpoint:    aws.String("http://sqs-sns.mnq.fr-par.scw.cloud"),
+		Endpoint:    aws.String("https://sqs.mnq.fr-par.scw.cloud"),
```

Comment:
nit: I'm pretty sure this endpoint is deprecated, all new MnQ endpoint should be in `*.scaleway.com`

Reply:
I'll let `*.scw.cloud` for now because we would need to recreate the queue to change the endpoint. Without recreating the queue, we are facing the following error:
`Post "https://sqs.mnq.fr-par.scaleway.cloud/": dial tcp: lookup sqs.mnq.fr-par.scaleway.cloud on 127.0.0.53:53: no such host`